src/sentry/sentry_apps/api/endpoints/sentry_app_authorizations.py

```diff
@@ -86,6 +87,8 @@ def post(self, request: Request, installation) -> Response:
                 },
             )
             return Response({"error": e.msg or "Unauthorized"}, status=403)
+        except UnableToAcquireLock:
```

Comment:
Would this happen frequently? I wonder if we need to do a frontend change to improve user experience if double requests to this endpoint can easily happen.